Src/library/net/abstract/socket/socket.e

```diff
@@ -1041,42 +1126,6 @@ feature {NONE} -- Implementation
 		deferred
 		end
 
-feature -- Socket Recv and Send timeout.
```

Comment:
If the other routines where in descendant, was it because they do not apply let's say for UNIX_SOCKET for example?